app/container-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eu
+
+bin='docker-gen'
+
+# run command if it is not starting with a "-" and is an executable in PATH
+if [ "${#}" -le 0 ] || \
+   [ "${1#-}" != "${1}" ] || \
```

Comment:
If you pass `--` to `command`, it will prevent `$1` from being interpreted as an option. That would make it possible to eliminate this check. (I don't think it's a problem accidentally running a command named something like `-x`; such commands shouldn't exist, and if they do then running it is no worse than accidentally running any other command that was intended to be a command-line option to `docker-gen`.)